packages/tasks/src/model-libraries.ts

```diff
@@ -670,6 +670,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.transformersJS,
 		filter: true,
 	},
+	trl: {
+		prettyLabel: "TRL",
+		repoName: "trl",
+		repoUrl: "https://github.com/huggingface/trl",
+		docsUrl: "https://huggingface.co/docs/trl",
```

Comment:
FYI I saw the other HF libs had `https://huggingface.co/docs/hub/{LIB_NAME}` but this URL doesn't exist for TRL, so I picked the one that does exist

Reply:
let's create a Hub doc page for TRL?

Reply:
The doc in Hub docs is kinda a summary "how to use the lib with the Hub" + some references, which is different than the full documentation. Ex: https://huggingface.co/docs/hub/transformers-js

Reply:
OK I'll add a Hub docs page first!

Reply:
PR here: https://github.com/huggingface/hub-docs/pull/1426